docs/en/latest/admin-api.md

```diff
@@ -657,12 +659,32 @@ After the execution is successful, nodes will not retain the original data, and
 
 ```
 
+Example 2: Create upstream and configure the scheme as `https`.
+
+```shell
+# Create upstream and configure the scheme as `https`
```

Comment:
We need a complete example, from apisix configuration to client access

Reply:
@Firstsawyou any update？

Reply:
Please check other desc too.

Reply:
Okay, I have checked for updates.